.gitpod.yml

```diff
@@ -47,6 +47,21 @@ tasks:
     before: pre-commit install --install-hooks
     init: leeway exec --filter-type go -v -- go mod verify
     openMode: split-right
+  - name: GoLand indexing
+    init: |
+      if [ ! "$GITPOD_HEADLESS" = "true" ] ; then exit ; fi
```

Comment:
Given this is quite a heavy-weight script, does it make sense to check it in as a script and reference in `.gitpod.yml`?

Reply:
sure, let's try fist whether we can make it actually work 😄